webapp/src/scenes/dashboard/components/WorkspaceCard.js

```diff
@@ -0,0 +1,169 @@
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+
+// material-ui components
+import { withStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import { ListItem, Typography } from '@material-ui/core';
+
+
+import WorkspaceCardActions from './WorkspaceCardActions';
+
+
+import * as Parser from '../../../services/handler/parser';
+import * as ReduxUtils from '../../../services/handler/reduxUtils';
+
+import {
+  administrationApi,
+  getDefaultApiCallback,
+  toastErrorMessage
+} from '../../../services/client/ml-lab-api';
+
+const styles = (theme) => ({
```

Comment:
I think the styles object still contains some data that was copied from ProjectCard.js but is not used here.